package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "email-alert-frontend",
+  "version": "1.0.0",
+  "description": "A frontend for creating and managing email subscriptions.",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs",
+    "lib": "lib"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "eslint": "^7.3.1"
```

Comment:
Should this actually be a dependency on [standard](https://www.npmjs.com/package/standard)?